fastchat/serve/monitor/monitor.py

```diff
@@ -293,7 +293,7 @@ def highlight_max_rank(s):
     return df.apply(highlight_max_rank, axis=1)
 
 
-def get_arena_table(arena_df, model_table_df, arena_subset_df=None):
+def get_arena_table(arena_df, model_table_df, arena_subset_df=None, round_digit=2):
```

Comment:
do we need `round_digit=2`?

Reply:
Ends up we don't use `round_digit` at all. Removed it.